packages/loader/container-loader/src/container.ts

```diff
@@ -1113,13 +1116,14 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
      * @param specifiedVersion - one of the following
      *   - undefined - fetch latest snapshot
      *   - otherwise, version sha to load snapshot
-     * @param pause - start the container in a paused state
+     * @param create - create the container when there is no snapshot.
```

Comment:
Maybe call it createOnDemand or createIfNonExisting?
Snapshot reference is really not on point here (it's side-effect of creating on demand on container.load() path).